lib/block_view_validator.go

```diff
@@ -2297,6 +2124,14 @@ func (bav *UtxoView) CreateUnjailValidatorTxindexMetadata(
 	return &UnjailValidatorTxindexMetadata{}, affectedPublicKeys
 }
 
+func SumValidatorEntriesTotalStakeAmountNanos(validatorEntries []*ValidatorEntry) *uint256.Int {
+	totalStakeAmountNanos := uint256.NewInt()
+	for _, validatorEntry := range validatorEntries {
```

Comment:
Is the assumption that the caller of this function is responsible for defining the validators to include? This function does not take validator status into consideration. We should add a comment on top of this function.